public/bower.json

```diff
@@ -18,5 +18,8 @@
     "paper-tabs": "Polymer/paper-tabs#~0.5.5",
     "paper-spinner": "Polymer/paper-spinner#~0.5.5",
     "webcomponentsjs": "WebComponents/webcomponentsjs#~0.5.5"
+  },
+  "resolutions": {
+    "webcomponentsjs": "^0.6.0"
```

Comment:
Can we get rid of this resolution by depending on ^0.6.0 instead of ~0.5.5 above?